target/linux/ramips/dts/mt7620a_zyxel_emg1702-t10a.dts

```diff
@@ -0,0 +1,58 @@
+#include "mt7620a_tbs.dtsi"
+
+/ {
+	compatible = "zyxel,emg1702-t10a", "ralink,mt7620a-soc";
+	model = "ZyXEL EMG1702-T10A";
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
```

Comment:
This can probably go faster ...

Reply:
Yes, I think so too... This is a standard MX25L6405D ... In the datasheet, it can go up to 33MHz... I used the same frequency from the mt7620a evaluation board, but I'll test with 30MHz...